dev/docker/README.md

```diff
@@ -0,0 +1,86 @@
+Apache REEF (incubator) Docker Tests
+====================================
+
+[Apache REEF (incubator)](http://reef.incubator.apache.org/) is 
+an approach to simplify and unify the lower layers of big data systems
+on modern resource managers. This project maintains docker images
+to simulate the target underlying systems as similar as possible.
+
+Docker-based Test Cluster
+-------------------------
+
+| OS           | Java            | Hadoop Version            | Description   | alias     |
```

Comment:
Oh, I love to do that. But, there is some limitation. Could you see the following repository?

https://github.com/dongjoon-hyun/docker-reef-tests

When I added more columns before, the table looks ugly. If you want, I will split the table into two. Is that okay?